code/hud/hudmessage.cpp

```diff
@@ -923,20 +822,69 @@ void hud_scrollback_button_pressed(int n)
 	}
 }
 
+// breaks up Msg_scrollback_vec into individual lines that can be rendered
+// to the screen one at a time. Also helps with figuring out the actual
+// scroll height of the scrollback UI.
+void hud_initialize_scrollback_lines()
+{
+
+	Msg_scrollback_lines.clear();
+
+	if ((Msg_scrollback_vec.size() > 0) && HUD_msg_inited) {
+
+		for (int j = 0; j < (int)Msg_scrollback_vec.size(); j++) {
+			line_node node_msg = Msg_scrollback_vec[j];
+
+			int width = 0;
+			int height = 0;
+			gr_get_string_size(&width, &height, node_msg.text.c_str(), (int)node_msg.text.length());
+
+			int max_width = Hud_mission_log_list2_coords[gr_screen.res][2];
+			if (width > max_width) {
+				char c_text[HUD_MSG_LENGTH_MAX];
+				strcpy_s(c_text, node_msg.text.c_str());
+
+				char* text = c_text;
+				char* split = nullptr;
+
+				split = split_str_once(text, max_width);
```

Comment:
I think declaration and first assignment can be merged here
```suggestion
				char* split = split_str_once(text, max_width);
```

Reply:
and done